Initial code to authorize and send an email from cmdline.

shalearkane opened the pull request:
Extra features : get credentials from file to prevent leaking on git repo.
Changes from official instructions : use SSL instead of TLS.

shalearkane committed:
Initial code to authorize and send an email from cmdline.
Extra features : get credentials from file to prevent leaking on git repo.
Changes from official instructions : use SSL instead of TLS.

srijan2002 reviewed (commented):
PR Labelled as hacktoberfest-accepted ,  also complete the Receiving Mail function

shalearkane committed:
Changes : added a file picker which can choose multiple files.

shalearkane committed:
Additions : created a new class for retrieving mail.
Changes : moved getCredentials to a new class
Improvement : removed the unneeded String results[] and made the no. of emails to be retrieved be known from the function call.
